.github/workflows/publish.yml

```diff
@@ -8,7 +8,31 @@ on:
             - v*
 
 jobs:
-    build:
+
+    build_wheels:
+
+        name: Build wheels on ${{ matrix.os }}
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                os: [ubuntu-22.04, windows-2022, macos-11]
+
+        steps:
+            - uses: actions/checkout@v3
```

Comment:
The `build_sdist` contains this step too, please update the `checkout` version to v3 there as well.

Reply:
Done.